### PR TITLE
feat: trusted nodes and script to generate the extended trusted nodes

### PR DESCRIPTION
**Pull request by vojtechsimetka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,27 @@
 # README.md
+
```
Comment (Eknir) [inline README.md]: no need to make changes in this file

File: src/trusted_nodes.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
+[
```
Comment (Eknir) [inline src/trusted_nodes.json]: make this initial trusted nodes and what you call `extended trusted nodes` into `trusted nodes` (the filename)

**Review by Eknir — APPROVED** [review]
LGTM, good work!

**Review by filoozom — APPROVED** [review]
🚀

**Pull request merged**